newton/_src/actuators/actuator.py

```
@@ -61,14 +71,52 @@ class Actuator:
 
         delay_state: Delay.State | None = None
         """Delay buffer state, or ``None`` if no delay is used."""
-        controller_state: Controller.State | None = None
-        """Controller-specific state, or ``None`` if stateless."""
+        drive_state: DriveBase.State | None = None
+        """Drive-specific state, or ``None`` if stateless."""
+
+        def __init__(
+            self,
+            delay_state: Delay.State | None = None,
+            drive_state: DriveBase.State | None | object = _DEPRECATED_UNSET,
```

Ruff (0.16.2)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)